src/view/uniprotkb/components/GeneNamesView.tsx

```diff
@@ -18,6 +18,20 @@ const GeneNamesView: React.FC<GeneNamesDataProps> = ({
   return <NameView {...props} />;
 };
 
+export const GeneNamesViewFlat = ({
+  name,
+  alternativeNames = [],
```

Comment:
Wondering if it would be worth adding a default to `GeneNamesListView` for consistency